src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
-* Finally, use Lighthouse CI on pull requests to ensure there are no regressions in Core Web Vitals before you deploy a change to production.
+[CrUX](https://developers.google.com/web/tools/chrome-user-experience-report/) is a [BigQuery dataset](https://developers.google.com/web/tools/chrome-user-experience-report/bigquery/getting-started) of field data gathered from [a segment of real Google Chrome users](https://developers.google.com/web/tools/chrome-user-experience-report/#methodology) from millions of websites. It also presents Core Web Vitals for websites with sufficient traffic, but only at the origin level. However, the [CrUX API](/chrome-ux-report-api/) can provide field data at the URL level, provided a URL has enough samples in the CrUX dataset.
 
-With that introduction, let's dive into the specific updates for each tool!
+#### When to use CrUX
+
+Even if you gather your own field data, CrUX is still useful. Though CrUX represents a subset of Chrome users, it's helpful to compare your website's field data to see how it aligns with its CrUX data. If you currently don't gather _any_ field data for your website, CrUX is especially valuable&mdash;provided your website is represented in its dataset.
+
+#### When _not_ to use CrUX
+
+Websites that don't receive enough traffic are not represented in the CrUX dataset. If this is the case for you, you'll need to gather your own field data to understand how your website performs in the field, as CrUX won't be an option.
+
+{% Aside 'caution' %}
+Since the data CrUX provides is a rolling average over the previous 28 days, it's not an ideal tool during development, as it will take a fair amount of time for improvements to be reflected in the CrUX dataset.
+{% endAside %}
+
+### PageSpeed Insights (PSI)
+
+[PSI](https://pagespeed.web.dev/) is a tool that reports lab data from CrUX _and_ field data from Lighthouse for a given page.
+
+#### When to use PSI
+
+PSI is great for assessing performance at the page level. It's a good choice for those in non-developer roles who wish to assess how specific pages are performing in the field, which is valuable if you're not gathering your own field data. PSI also provides recommendations for non-developers who are not already using Lighthouse, and from a more consistent baseline than running Lighthouse from your own laptop.
+
+#### When _not_ to use PSI
+
+PSI may not be worth using for developers already using Lighthouse. From the field data perspective, it's not a fit for websites that aren't represented in the CrUX dataset.
+
+### Search Console
+
+[Search Console](https://search.google.com/search-console/about) measures your site's search traffic and performance, including Core Web Vitals. A valuable feature of Search Console is that it assesses groups of similar pages (for example, pages that use the same template). Search Console also includes a Core Web Vitals report based on field data from CrUX.
```

Comment:
We probably want to link to Search Console's "Core Web Vitals Report" docs somewhere in this paragraph: https://support.google.com/webmasters/answer/9205520